plugin.rb

```diff
@@ -8,38 +8,39 @@
 
 courses = Hash.new
 courses = {
+    11 => "e11_scratch_00",
```

Comment:
Both Scratch courses should have URLs like this: https://scratch.mit.edu/projects/[PROJECT_ID_NUMBER]/